### ISSUE TITLE
failure on adding new user manually from dashboard is missing feedback

### ISSUE THREAD

**Issue by Huilensolis** [supabase/supabase#23552]
# Bug report

## Describe the bug

if you try to create a new user from the dashboard and it fails, there is no feedback indicating that it went wrong.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to the supabase auth dashboard `https://supabase.com/dashboard/project/${projectId}/auth/users`
![image](https://github.com/supabase/supabase/assets/113150193/29d02563-48a7-436b-bcb9-6224e96279a4)

3. Click on "Add new user"
![image](https://github.com/supabase/supabase/assets/113150193/f062e41d-29cc-402f-84ac-5805f0c99bd3)

5. click on "Create new user"
![image](https://github.com/supabase/supabase/assets/113150193/e0532bd7-3978-4343-a0ea-da2a85221fee)

7. fill the form
8. click "Create user"
9. if it fails, it won't show any UI showing an error

## Expected behavior

Show a toast with the API response message

**Comment by encima** [comment]
Hey @Huilensolis 

Thanks for opening! I assume you know this failed due to the Developer Console? What error do you see? Just to help us reproduce?

Thanks!

**Comment by GaryAustin1** [comment]
Are you using Brave browser?  I've seen users with the popups not showing there.
Also in the Postgres or auth logs is usually the error and 90% of the time an auth.users trigger function failing.

**Comment by Huilensolis** [comment]
> Are you using Brave browser? I've seen users with the popups not showing there. Also in the Postgres or auth logs is usually the error and 90% of the time an auth.users trigger function failing.

yes, I'm using brave

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/51e478a8-fc44-4106-a498-2318e80c9a9c)

**Comment by Huilensolis** [comment]
> Hey @Huilensolis
> 
> Thanks for opening! I assume you know this failed due to the Developer Console? What error do you see? Just to help us reproduce?
> 
> Thanks!

I thought that I didn't need to include the "steps to reproduce" because there was just no catch case, but I have realized is not the only possible case, maybe its a bug on the toast and it indeed should pop up but it is not. mb

In my case, it was a Postgres error thrown by a function triggered by "insert" into auth.users, it failed because I was accessing an inexisting key

the function:
```SQL
begin
  insert into public.user (id, username, full_name, avatar_url)
  values (new.id, new.raw_user_meta_data->>'try_to_acces_to_undefined_key', new.raw_user_meta_data->>'full_name', new.raw_user_meta_data->>'avatar_url');
  return new;
end;
```

and to trigger it I used the SQL editor:
```sql
create trigger on_auth_user_created
after insert on auth.users for each row
execute procedure handle_new_user ();
```

**Comment by Huilensolis** [comment]
> ![image](https://private-user-images.githubusercontent.com/54564956/327625774-51e478a8-fc44-4106-a498-2318e80c9a9c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ2OTk1MTcsIm5iZiI6MTcxNDY5OTIxNywicGF0aCI6Ii81NDU2NDk1Ni8zMjc2MjU3NzQtNTFlNDc4YTgtZmM0NC00MTA2LWE0OTgtMjMxOGU4MGM5YTljLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAzVDAxMjAxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI2NzkwOGQ5ZWZlMDY1MTVmNjAyNzNkMGVmYWRkYzgwMmQwZDk3MTgyMzVjNTRkNWNkOGNmODA0ZmNjNTQ3ODEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SL8T3PWwvRUYUQZFaNAs8SNk-3kUJ02e4akcWWyZLJw)

they have shown up on previous occasions when a Postgres error was thrown. i don't think it may be the problem

**Comment by GaryAustin1** [comment]
So is your real code problem resolved with the auth.users trigger?
And you are just reporting a browser bug with the popup? 
That popup shows up in many other browsers for this type of error.

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/1ff16f4f-9e2b-4d3d-9dea-df4ddac47ca8)
The popup itself is usually not that useful, but at least there is indication of error. 
And for that operation is almost always auth.users trigger function.

**Comment by Huilensolis** [comment]
> So is your real code problem resolved with the auth.users trigger? And you are just reporting a browser bug with the popup? That popup shows up in many other browsers for this type of error.

yes, the real code problem is solved now. I'm just reporting no popup in that case, as I thought they used a optimistic UI, as it creating new users is something supposed to success. so I thought they had no popup for a catch case

**Comment by GaryAustin1** [comment]
Yes, it is a real bug that needs reporting as far as browser, mainly as you did not provide any info on the browser was bringing that up as I've seen that helping others limited to Brave.

Glad you found the underlying issue.

**Comment by GaryAustin1** [comment]
Or it is a feature of Brave to block popups.. I don't know.

**Comment by Huilensolis** [comment]
![image](https://github.com/supabase/supabase/assets/113150193/641f7fc2-f621-4607-a278-b54a7ed6eb89)
I tried to create an account with an already-used email and it failed. If Brave blocks the pop-up, it should show an icon on the URL, right?

**Comment by GaryAustin1** [comment]
Sorry, I don't know, I'm not a UI or SB dev.  Just really pointing out Brave/SB has this issue from others I've helped and in case you had not found your underlying issue what it likely was and where to look.

**Comment by Huilensolis** [comment]
It is a brave compatibility issue then. if this issue is duplicated, they can close it, thank you for your help @GaryAustin1 !

**Comment by PLyn** [comment]
@GaryAustin1 I can confirm that strict adblocking blocks the toast messages from being seen. Reproduced in chrome with ublock origin

**Comment by Hallidayo** [comment]
Hi all, We’re going to close this one due to inactivity. Please feel free to reopen if you’re still having issues here or open a new issue.
